src/libraries/Microsoft.PowerFx.Core/Entities/External/TableDelegationInfo.cs

```diff
@@ -266,6 +268,33 @@ public SelectionRestrictions()
         }
     }
 
+    public class SummarizeCapabilities
+    {
+        public virtual bool IsSummarizableProperty(string propertyName)
+        {
+            return false;
+        }
+
+        public virtual bool IsSummarizableMethod(SummarizeMethod method)
+        {
+            return false;
+        }
+
+        public SummarizeCapabilities()
+        {
+        }
+    }
+
+    public enum SummarizeMethod
```

Comment:
>SummarizeMethod [](http://example.com/codeflow?start=16&length=15)

fwiw - see IAggregator. 
This is fine now. but I wonder if we'll need to distinguish between types (MinNumber vs. MinDate) .